.github/workflows/backend.yml

```diff
@@ -33,7 +38,56 @@ jobs:
           go-version: '1.21.5' 
 
       - name: run tests
-        run: go test ./... 
+        run: go test -race -coverprofile=coverage.out -covermode=atomic ./...
+
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v4.0.1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          slug: incompetent-hosting-provider/monorepo
```

Comment:
Check in codecov please. I think this will overwrite its own values.
Instead set the slug to backend in this case and cli,... in the other cases

Reply:
![image](https://github.com/incompetent-hosting-provider/monorepo/assets/76261866/10399d75-afec-4839-a8fc-6df381bd143a)

Are you sure that it will overwrite? Because the dashboard shows cli as well as backend coverage. With the other slug it didn't worked for me. And for the Terraform the tests are failing because of that binary error. therefore no upload is triggered.

But maybe I didn't understand it correctly.

Reply:
hmm maybe that changed, seems like you are right